LDAR_Sim/src/virtual_world/fugitive_emission.py

```diff
@@ -159,6 +162,22 @@ def tag_leak(
         self._tagging_rep_delay = tagging_rep_delay
         return True
 
+    def calc_mitigated(self) -> float:
```

Comment:
Can the simulation end data just be passed through calc_mitigated instead of stored from initialization

Reply:
Good catch. Don't know why i didn't see this.